### ISSUE TITLE
Support need to retrieve ISIS setbit in ATE

### ISSUE THREAD

**Issue by cprabha** [openconfig/featureprofiles#1176]
Hi,

We need to have support to retrieve ISIS setbit in ATE.
Below code is not working , returns null value.

gnmi.GetAll(t, ate, lsps.Tlv(oc.IsisLsdbTypes_ISIS_TLV_TYPE_EXTENDED_IPV4_REACHABILITY).ExtendedIpv4Reachability().PrefixAny().SBit().State())

Thanks,
Prabha

**Comment by cprabha** [comment]
Thanks Greg.

**Comment by greg-dennis** [comment]
@cprabha, I don't think you meant to query the `SBit` path above. It seems you meant the "set-bit" within the overload-bit. Some existing tests create paths to that, like here:
https://github.com/openconfig/featureprofiles/blob/main/feature/experimental/isis/ate_tests/lsp_updates_test/lsp_updates_test.go#L52

As of today, it is only queried on the DUT. I am checking on ATE support. When I checked before, based on the original comment, I had checked on "s-bit" support, not overload-bit/set-bit support.
